instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/cache/weaklockfree/DetachedThreadLocal.java

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
A couple of unused files, even when vendoring we may as well leave out completely unused ones I guess. Probably easier to copy from here actually :P

https://github.com/open-telemetry/opentelemetry-java/tree/main/context/src/main/java/io/opentelemetry/context/internal/shaded